### DISCUSSION TITLE
Is there a way to create a Run Button for running the test. The default button is really small and invisible

### DISCUSSION THREAD

**Discussion by pratikgcet** [codesandbox/sandpack#820] [Q&A]
Looking for a way to create a Run Button for running the test. The default button is really small and invisible.
I need to have a button in "verbose" mode

**Comment by danilowoz** [comment]
Hey, @pratikgcet. Thanks for opening this question. 

Currently, this button is not customizable, and it follows the guidelines of the Sandpack design system. Would you like to propose a new button or a way to extend it? I'd be more than happy to guide you on a PR to make it happen.

**Reply by pratikgcet** [reply]
Thanks @danilowoz . Yes I would propose extending the useSandpackNavigation or an additional hook to run test using a customisable button please. Please guide me on the PR. I would love to get it done quickly as it's a show stopper for us, not being able to have a button to run the tests

**Comment by NiranjanKumar001** [comment]
is it possible to remove the run button as i managed to make run button as per mine requirement but i dont know how can i remove the default one is there any way if possible pls reply its urgent?
